src/Servers/Kestrel/Core/src/KestralBadHttpRequestException.cs

```diff
@@ -88,9 +88,6 @@ internal static BadHttpRequestException GetException(RequestRejectionReason reas
                 case RequestRejectionReason.InvalidHostHeader:
                     ex = new BadHttpRequestException(CoreStrings.BadRequest_InvalidHostHeader, StatusCodes.Status400BadRequest, reason);
                     break;
-                case RequestRejectionReason.UpgradeRequestCannotHavePayload:
```

Comment:
Nit: can we fix the file name?

Reply:
😆 "Kestral" is how you pronounce it when you want to sound fancy.